src/components/comments/CommentList.js

```diff
@@ -2,20 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import RetrieveCommentForm from './RetrieveCommentForm';
 
-export const CommentList = ({ comments }) => (
+export const CommentList = ({ comments, article }) => (
   <div>
     {comments.map(comment => (
-      <RetrieveCommentForm key={comment.id} comment={comment} />
+      <RetrieveCommentForm key={comment.id} comment={comment} article={article} />
```

Comment:
Nice work @MuhanguziDavid,

Is it not possible to pass the comment alone and retrieve the id from it in the `RetrieveCommentForm` component?
That way there is no need to pass in a key that already exists in the comment

Reply:
Yes, @RonKbS , it is possible. Changes have been made accordingly.